package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "10.0.1",
+  "version": "10.1.1",
```

Comment:
When you increment a new minor or major version, all of the other numbers are reset to 0. 

https://semver.org/#spec-item-7 & https://semver.org/#spec-item-8